pypreprocess/nipype_preproc_spm_utils.py

```diff
@@ -663,6 +663,144 @@ def _do_subject_segment(subject_data, output_modulated_tpms=True, spm_dir=None,
     return subject_data.sanitize()
 
 
+def _do_subject_newsegment(subject_data, output_modulated_tpms=True,
+                           spm_dir=None, matlab_exec=None, spm_mcr=None,
+                           normalize=False, caching=True, report=True,
+                           software="spm", hardlink_output=True):
+    """
+    Wrapper for running spm.NewSegment with optional reporting.
+
+    If subject_data has a `results_gallery` attribute, then QA thumbnails will
+    be commited after this node is executed
+
+    Parameters
+    -----------
+    subject_data: `SubjectData` object
+        subject data whose anatomical image (subject_data.anat) is to be
+        segmented
+
+    output_modulated_tpms: bool, optional (default False)
+        if set, then modulated TPMS will be produced (alongside unmodulated
+        TPMs); this can be useful for VBM
+
+    caching: bool, optional (default True)
+        if true, then caching will be enabled
+
+    normalize: bool, optional (default False)
+        flag indicating whether warped brain compartments (gm, wm, csf) are to
+        be generated (necessary if the caller wishes the brain later)
```

Comment:
This is a flag related to `spm.Segment`. It doesn't hold for `spm.NewSegment`: to be removed.